README.md

```diff
@@ -72,3 +72,96 @@ Earlier versions may also work, but come with no guarantees.
 
 Usage Examples
 --------------
+
+**Note:** This module is fully compatible with Node/Babel/ES6/ES5. Simply
+omit the type declarations when using a language other than TypeScript.
+
+**Kernel**
+
+```typescript
+import {
+  IKernelFuture, IKernelId, IKernelInfo, IKernelMsg, Kernel
+} from 'jupyter-js-services';
+
+// get a list of available sessions
+var listPromise = Kernel.list('http://localhost:8000');
+
+// start a specific session
+var kernel = NotebookSession('http://localhost:8000');
+listPromise.then((kernelList: IKernelId[]) => {
```

Comment:
You don't need to explicitly type the `kernelList` arg. The promise carries the type, so the function will be contextually-typed automatically by the compiler. 

```typescript
listPromise.then(kernelList => {
  kernel.start(kernelList[0]);
});
```